docs/examples/coreos-kops-tests-multimaster.md

```diff
@@ -0,0 +1,310 @@
+# USING KOPS WITH COREOS - A MULTI-MASTER/MULTI-NODE PRACTICAL EXAMPLE
+
+## WHAT WE WANT TO ACOMPLISH HERE ?.
+
+The exercise described on this document will focus on the following goals:
+
+- Demonstrate how to use a production-setup with 3 masters and multiple working nodes (two).
+- Change our default base-distro (Debian 8) for CoreOS stable, available too as an AMI on AWS.
+- Ensure our masters are deployed on 3 different AWS availability zones.
+- Ensure our nodes are deployed on 2 different AWS availability zones.
+
+
+## PRE-FLIGHT CHECK:
+
+Before rushing in to replicate this exercise, please ensure your basic environment is correctly setup. See the KOPS tutorial for more information: [KOPS Tutorial](https://github.com/kubernetes/kops/blob/master/docs/aws.md). Ensure that the following is already working in your environment:
+
+- AWS cli fully configured (aws account already with proper permissions/roles needed for kops). Depending on your distro, you can setup directly from packages, or if you want the most updated version, use "pip" and installed by issuing a "pip install awscli" command. Your choice !.
+- Local ssh key ready on ~/.ssh/id_rsa / id_rsa.pub. You can generate it using "ssh-keygen" command: `ssh-keygen -t rsa -f ~/.ssh/id_rsa -P ""`
+- Region set to us-east-1 (az's: us-east-1a, us-east-1b, us-east-1c, us-east-1d and us-east-1e). For this exercise we deployed our cluster on US-EAST-1. If you want to use another region, feel free to change it. Just ensure to have at least 3 availability zones.
```

Comment:
Instead of "Just ensure to have at least 3 availability zones" how about "It is recommended to pick a region with at least 3 availability zones, for true high-availability".

Some regions only have 2 AZs; we support multi master in those regions, but it isn't as HA.